yapf/yapflib/style.py

```diff
@@ -17,6 +17,8 @@
 import re
 import textwrap
 
+import toml
```

Comment:
[help wanted] I added external dependency because there's no toml loader in standard libraries. Is this OK?

Reply:
I'd prefer this to be seamless. So if someone doesn't have the `toml` package installed, then we just skip this support all together. Is it possible to import the `toml` conditionally and if it fails to import set a flag or something?

Reply:
I agree with you! Fixed this in 58b9034